cafebabel/templates/home.html

```diff
@@ -21,5 +21,16 @@
   {{ meta('name=twitter:url', request.url) }}
 {% endblock %}
 
-
-{% block body %}{% endblock %}
+{% block body_class %}home{% endblock %}
+{% block body %}
+  {% for article in articles %}
+    {% if loop.first %}
+      <h4>Editor's pick</h4>
```

Comment:
Not fond of adding an `h4` here if there is no `h3` before. There should be no gap between `hX` hierarchy (both accessibility and SEO concerns)

Reply:
`h3` are the article titles. So i chose `h2`